test/units/testsuite-46.service

```diff
@@ -3,8 +3,8 @@
 Description=TEST-46-HOMED
 Wants=getty-pre.target
 Before=getty-pre.target
-Wants=systemd-homed.service
-After=systemd-homed.service
+Requires=systemd-homed.service systemd-userdbd.service
+After=systemd-homed.service systemd-userdbd.service
```

Comment:
Not sure, but these services are triggered by bus call, and not long-running services.
Why they need to be started before the test service??

Reply:
E.g. IIRC, tests for localed or logind does not require or want these services started before the test service. If these are necessary, then I guess something broken, though I am not sure whether the test script is broken or the service itself is broken.

Reply:
That's because we don't call `systemctl preset-all` on the test image. In the userdbd case the `systemd-userdbd.socket` is disabled, so the respective service is never started, in case of homed there's no symlink to `dbus-org.freedesktop.home1.service` as that's created during `systemctl enable`.

I thought about doing `preset-all` after we create the test image (and before we call `mask_supporting_services()` to avoid stuff like this), but I'm not sure if it wouldn't break things.